todo_app/Selenium Tests/test_Selenium.py

```diff
@@ -0,0 +1,47 @@
+import os, pytest, dotenv
+from threading import Thread
+from todo_app.data.trello_items import create_trello_board, delete_trello_board
+from todo_app import app
+from selenium import webdriver 
+from selenium.webdriver.support.ui import Select
+
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    board_id = create_trello_board('Test_Board')
+    file_path = dotenv.find_dotenv('.env')
+    dotenv.load_dotenv(file_path)
+    os.environ['TRELLO_API_BOARD_ID'] = board_id
+
+    application = app.create_app()
+    
+    thread = Thread(target=lambda: application.run(use_reloader=False))
+    thread.daemon = True
+    thread.start()
+    yield app
+    
+    thread.join(1)
+    delete_trello_board(board_id)
+
+@pytest.fixture(scope="module")
+def driver():
+    with webdriver.Edge(("C:\Program Files (x86)\Microsoft\Edge\Application\msedgedriver.exe")) as driver:
```

Comment:
Really we should place the driver on your PATH, and then this argument can be empty (rather than insisting every developer have the driver in exactly the same file directory)